modules/local/get_software_versions.nf

```diff
@@ -21,13 +21,54 @@ process GET_SOFTWARE_VERSIONS {
     path versions
 
     output:
-    path "software_versions.tsv"     , emit: tsv
-    path 'software_versions_mqc.yaml', emit: yaml
+    path "software_versions.yml"    , emit: yml
+    path 'software_versions_mqc.yml', emit: mqc_yaml
 
-    script: // This script is bundled with the pipeline, in nf-core/rnaseq/bin/
+    script:
     """
-    echo $workflow.manifest.version > pipeline.version.txt
-    echo $workflow.nextflow.version > nextflow.version.txt
-    scrape_software_versions.py &> software_versions_mqc.yaml
+    #!/usr/bin/env python
+
+    import yaml
+    from textwrap import dedent
+
+    def _make_versions_html(versions):
+        html = ["<dl>"]
+        for process, tmp_versions in versions.items():
+            for tool, version in tmp_versions.items():
+                html.append(
+                    dedent(
+                        f'''\\
+                        <dl>
+                            <dt>{process}</dt>
+                            <dt>{tool}</dt>
+                            <dd>{version}</dd>
+                        </dl>
+                        '''
+                    )
+                )
+        html.append("</dl>")
+        return "\\n".join(html)
```

Comment:
```suggestion
    def _make_versions_html(versions):
        html = ["""
        <style>
        tbody:nth-child(even) {
            background-color: #f2f2f2;;
        }
        </style>
        <table style="width:100%">
            <thead>
                <tr>
                    <th> Process Name </th>
                    <th> Software </th>
                    <th> Version  </th>
                </tr>
            </thead>
        """]
        for process, tmp_versions in versions.items():
            html.append("<tbody>")
            print_module = 0
            for tool, version in tmp_versions.items():
                print_module += 1
                html.append(
                    dedent(
                        f'''\
                        <tr>
                            <td><samp>{process if (print_module == 1) else ''}</samp></td>
                            <td><samp>{tool}</samp></td>
                            <td><samp>{version}</samp></td>
                        </tr>
                        '''
                    )
                )
            html.append("</tbody>")
        html.append("</table>")
        return "\\n".join(html)
```
Can maybe try this here before black complains. Is it working well?

Reply:
Fixed some quotes...